src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -140,30 +140,31 @@ extern "C" {
 	}
 
 	NODELOOPPLUGIN_DECL int ibmras_monitoring_plugin_init(const char* properties) {
+		uv_check_init(uv_default_loop(), &check_handle);
+		uv_unref(reinterpret_cast<uv_handle_t*>(&check_handle));
+
+		plugin::timer = new uv_timer_t;
+		uv_timer_init(uv_default_loop(), plugin::timer);
+		uv_unref((uv_handle_t*) plugin::timer); // don't prevent event loop exit
+
 		return 0;
```

Comment:
It unrefs the handles, and doesn't start them, so I don't think it should have any negative effects.